stl/inc/utility

```diff
@@ -132,7 +132,7 @@ struct pair { // store a pair of values
     template <class _Uty1 = _Ty1, class _Uty2 = _Ty2,
         enable_if_t<conjunction_v<is_default_constructible<_Uty1>, is_default_constructible<_Uty2>>, int> = 0>
     constexpr explicit(
-        !_Is_implicitly_default_constructible<_Uty1>::value || !_Is_implicitly_default_constructible<_Uty2>::value)
+        !conjunction_v<_Is_implicitly_default_constructible<_Uty1>, _Is_implicitly_default_constructible<_Uty2>>)
         pair() noexcept(
             is_nothrow_default_constructible_v<_Uty1>&& is_nothrow_default_constructible_v<_Uty2>) // strengthened
         : first(), second() {}

```

Comment:
Thanks - these changes look correct. However, below, the `#if _HAS_CONDITIONAL_EXPLICIT` codepaths for `pair(const _Ty1& _Val1, const _Ty2& _Val2)` and `pair(_Other1&& _Val1, _Other2&& _Val2)` aren't being changed, but it seems they should all be consistent.

Reply:
Done